test/backend/routes.spec.js

```diff
@@ -198,58 +216,93 @@ describe('Routes - ', () => {
         .catch(done);
     });
 
-    it('index page - returns 200', function(done) {
-        GET('')
-        .then(res => {
-            assert.equal(res.status, 200);
-            done();
-        })
+    it('allows access to login page without logging in', function(done) {
+        GET('login')
+        .then(() => done())
         .catch(done);
     });
 
-    // Settings
-    it('settings/urls - returns 200 and the urls', function(done) {
-        GET('settings/urls')
-        .then(res => res.json().then(json => {
-            assert.equal(res.status, 200);
-            assert.equal(json.http, 'http://localhost:9494');
-            assert.equal(json.https, '');
-            done();
-        }))
+    it('allows access to connections page without logging in', function(done) {
```

Comment:
i don't quite understand this - is the "connections" page the page with all of the database credentials? shouldn't that page be authenticated?

Reply:
The url doesn't requires authentication, since it is the home page. But a user cannot see any connections without logging in (Since other requests to `/connections` etc are not allowed for un-authenticated users:


![image](https://user-images.githubusercontent.com/1546532/31029746-edde2e74-a520-11e7-9abd-362068c7b92c.png)

Reply:
We can probably have a better UI for the homepage for unauthenticated users though..